plugins/cxf/src/main/java/com/navercorp/pinpoint/plugin/cxf/interceptor/CxfLoggingMessageMethodInterceptor.java

```diff
@@ -49,6 +50,7 @@ protected CxfLoggingMessageMethodInterceptor(TraceContext traceContext, MethodDe
      * @param methodName the method name
      * @return the attribute
      */
+    @Deprecated
     protected <T> T getAttribute(Object object, String methodName) {
```

Comment:
I think we can safely remove this method instead of deprecating since the method has not been part of any official release yet.

Reply:
OK